File: Steven/src/main/java/parser/nodes/expression/LessThan.java
package parser.nodes.expression;

import parser.Visitor;
import parser.nodes.AbstractNode;

/**
 * Created by Steven Kok on 21/02/2015.
 */
public class LessThan extends LogicalOperator {
    public LessThan(Expression left, Expression right) {
        super(left, right);
    }

    @Override
    public AbstractNode accept(Visitor visitor) {
        return visitor.visit(this);
    }
}

File: FelipezConde/QuestionnaireLanguage/AST/Nodes/Expressions/Binaries/LessThan.cs
using AST.Nodes.Interfaces;

namespace AST.Nodes.Expressions.Binaries
{
    public class LessThan : Binary
    {
        public LessThan(Expression left, Expression right, PositionInText position)
            : base(left, right, position)
        { }

        public override T Accept<T>(VisitorInterfaces.IExpressionVisitor<T> visitor)
        {
            return visitor.Visit(this);
        }

        public override string ToString()
        {
            return "<";
        }
    }

}
